pkg/cableengine/healthchecker/pinger.go

```diff
@@ -0,0 +1,138 @@
+package healthchecker
+
+import (
+	"math"
+	"time"
+
+	"github.com/go-ping/ping"
+	"k8s.io/klog"
+)
+
+var waitTime time.Duration = 15
+var size uint64 = 1000
+
+type Pinger struct {
+	remoteEndpointName string
+	healthCheckIp      string
+	statistics         Statistics
+	Status             string
+	stopCh             chan struct{}
+	pinger             *ping.Pinger
+}
+
+type Statistics struct {
```

Comment:
Can we add some unit testing for the Statistics object?

Reply:
Done.